spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET /index' do
+    it 'returns http index success' do
+      get('/users')
+      expect(response).to have_http_status(:success)
+    end
+    it 'returns http index success' do
+      get('/users')
+      expect(response).to render_template(:index)
+    end
+    it 'renders the show template' do
+      get('/users')
+      expect(response.body).to include('Welcome to user index page')
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http show success' do
+      get '/users/:id', params: { id: 1 }
+      expect(response).to have_http_status(:success)
+    end
+    it 'returns render template Show' do
+      get '/users/:id', params: { id: 1 }
+      expect(response).to render_template(:show)
+    end
+    it 'returns render template Show' do
+      get '/users/:id', params: { id: 1 }
+      expect(response.body).to include('These are a user for a Id')
+    end
+  end
```

Comment:
- [ **OPTIONAL** ] Good job adding all the changes I required for the previous review. But I think it would be better if you update some messages of your tests, in the way they have the correct meaning of what they test. You can check all your tests and apply this approach for both files.
Example:
```ruby
 # Bad
  ...
  it 'returns render template Show' do
      get '/users/:id', params: { id: 1 }
      expect(response.body).to include('These are a user for a Id')
    end
    ...

 # Good
  ...
  it 'ensures the body to include the correct placeholder text.' do
      get '/users/:id', params: { id: 1 }
      expect(response.body).to include('These are a user for a Id')
    end
  ...
```

Reply:
Ok im going to change it